libmscore/style.cpp

```diff
@@ -1406,7 +1406,7 @@ const TextStyle textLineTextStyle {{
       { Sid::textLineFontBold,                   Pid::BEGIN_FONT_BOLD         },
       { Sid::textLineFontItalic,                 Pid::BEGIN_FONT_ITALIC       },
       { Sid::textLineFontUnderline,              Pid::BEGIN_FONT_UNDERLINE    },
-      { Sid::defaultAlign,                       Pid::ALIGN                  },
+      { Sid::textLineTextAlign,                  Pid::ALIGN                  },
       { Sid::defaultOffset,                      Pid::OFFSET                 },
```

Comment:
Why not to replace `defaultOffset` with `textLineTextOffset` as well? Could you please describe the purpose of these settings to keep it for history?

Reply:
Oops, didn't realise that there was a specific one for defaultOffset as well - I'll replace that. Basically, most elements have generic `defaultAlign` and `defaultOffset` Sids, but the TextLine element maps the `ALIGN` pid to the more specific `textLineTextAlign` Sid. It seems like someone just copied and pasted this section without realising that for a TextLine, the `ALIGN` Pid is mapepd to `textLineTextAlign`, not `defaultAlign`. Same for `OFFSET`.